src/components/Profile/Profile/AboutContainer/AboutContainerStyle.tsx

```diff
@@ -54,7 +52,7 @@ export const InfoLabel = styled.span`
   font-size: ${STYLE_CONSTANTS.FONT_SIZE.medium};
   color: ${STYLE_CONSTANTS.COLORS.lightGrey};
   @media (max-width: ${STYLE_CONSTANTS.SCREEN.laptop}) {
```

Comment:
add lines between the selectors for better readability

Reply:
fixed in all files